obi_generative_gui/alb.tf

```diff
@@ -0,0 +1,40 @@
+resource "aws_lb_target_group" "obi_generative_gui_private_tg" {
+  #ts:skip=AC_AWS_0492
+  name        = "obi-generative-gui-private"
+  port        = 8000
```

Comment:
I saw this 8000 port used in a few places, maybe worth to set it as an internal module variable?